include/aspect/velocity_boundary_conditions/gplates.h

```diff
@@ -44,6 +46,7 @@ namespace aspect
        * values at one particular time are interpolated between the two
```

Comment:
This comment is not longer true. We only keep one file around in GPlatesLookup, but GPlates has two GPlatesLookup objects and interpolate between them (no need to mention that here).